getting_started/openshift-overview.adoc

```diff
@@ -41,8 +41,10 @@ Develop and deploy containerized applications with {product-title}. {product-tit
 * **xref:../applications/projects/working-with-projects.adoc#working-with-projects[Work with projects]**: Create projects from the {product-title} web console or OpenShift CLI (`oc`) to organize and share the software you develop.
 
 * **xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Work with applications]**:
-Use the xref:../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective] in the {product-title} web console to
+
+Use the xref:../web_console/web-console-overview.adoc#about-developer-perspective_web-console-overview[*Developer* perspective] in the {product-title} web console to
```

Comment:
Consider "the *Developer* perspective" in the link title so that it is consistent with the two preceding modules.